src/main/java/uk/gov/hmcts/reform/iacasepaymentsapi/domain/entities/fee/OrganisationResponse.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.hmcts.reform.iacasepaymentsapi.domain.entities.fee;
+
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+
+@ToString
+@EqualsAndHashCode
+public class OrganisationResponse {
+
+    private String companyNumber;
+    private String name;
+
+    private OrganisationResponse() {
+
+    }
+
+    public OrganisationResponse(String companyNumber, String name) {
```

Comment:
We need to model Prof Data response correctly based on the real response feed

Reply:
Added name and account list